backoffice/config/settings/base.py

```diff
@@ -332,7 +332,7 @@
 # https://django-allauth.readthedocs.io/en/latest/configuration.html
 ACCOUNT_USER_MODEL_USERNAME_FIELD = None
 # https://django-allauth.readthedocs.io/en/latest/configuration.html
-ACCOUNT_EMAIL_VERIFICATION = False
+ACCOUNT_EMAIL_VERIFICATION = "none"
```

Comment:
Shouldn't this be `None`?

Reply:
Dont think so. On the documentation all options are strings https://docs.allauth.org/en/dev/account/configuration.html#:~:text=ACCOUNT_EMAIL_VERIFICATION

Reply:
👍